src/wmtk/multimesh/utils/find_local_switch_sequence.cpp

```diff
@@ -1,45 +1,132 @@
 #include "find_local_switch_sequence.hpp"
+#include <cassert>
 #include "local_switch_tuple.hpp"
 namespace wmtk::multimesh::utils {
 
 
 namespace {
+std::pair<bool, std::vector<PrimitiveType>> find_local_switch_sequence_on_edge(
```

Comment:
this is on another PR right? you should link / mention this PR in the other one

Reply:
Ah yes, I mentioned it on the issues linked to that pr.